### PR TITLE
:sparkles: hero section

### PR DESCRIPTION
**Pull request by MarioBatalha**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206109714345725

### CODE REVIEW

File: views/institutional/components/hero/index.tsx [hunk 1]
```diff
@@ -9,7 +9,50 @@ const Hero: FC = () => (
     backgroundPosition="bottom center"
     backgroundSize="cover"
   >
-    <Box mt="2xl">Hero</Box>
+    <Box
+      height="100vh"
+      width="100vw"
+      overflow="hidden"
+      bg="url(/img/hero-illustration.svg)"
```
Comment (KipandaJr) [inline views/institutional/components/hero/index.tsx]: try convert to `webp`

**Review by KipandaJr — CHANGES_REQUESTED** [review]

File: views/institutional/components/hero/index.tsx (hunk 1 above)
Comment (MarioBatalha) [reply views/institutional/components/hero/index.tsx]: Yes, I try to use webp but I don't like the image quality.
 SVG has a better quality than webp.

File: views/institutional/components/hero/index.tsx [hunk 2]
```diff
@@ -9,7 +9,50 @@ const Hero: FC = () => (
...
+          width={['100%', '100%', '100%', '40%']}
+          height={['100%', '100%', '100%', '35%']}
+        >
+          <Typography
+            textAlign="center"
+            as="h1"
+            whiteSpace="wrap"
+            fontSize={['2.5rem', '3.563rem']}
+            fontWeight="900"
+            lineHeight="4rem"
```
Comment (atilasousa) [inline views/institutional/components/hero/index.tsx]: check this size in the ui kit

File: views/institutional/components/hero/index.tsx [hunk 3]
```diff
@@ -9,7 +9,50 @@ const Hero: FC = () => (
...
+            textTransform="uppercase"
+            color="onSurface"
+            size="large"
+            variant="body"
+          >
+            Streamlined Development for Move Creators
+          </Typography>
+          <Button textTransform="uppercase" my="4rem" variant="filled">
```
Comment (atilasousa) [inline views/institutional/components/hero/index.tsx]: size check

**Review by atilasousa — COMMENTED** [review]
and very important please check the mobile version

Screenshot 2023-12-13 at 22 57 09

File: views/institutional/components/hero/index.tsx [hunk 4]
```diff
@@ -1,15 +1,59 @@
-import { Box } from '@interest-protocol/ui-kit';
+import { Box, Button, Typography } from '@interest-protocol/ui-kit';
 import { FC } from 'react';
 
 const Hero: FC = () => (
   <Box
     pt="10xl"
-    minHeight="100vh"
+    minHeight="100%"
     bg="url(/img/hero.png)"
```
Comment (KipandaJr) [inline views/institutional/components/hero/index.tsx]: all image file that u used on our site, u need to put like webp

File: views/institutional/components/hero/index.tsx [hunk 5]
```diff
@@ -1,15 +1,59 @@
...
     backgroundPosition="bottom center"
     backgroundSize="cover"
   >
-    <Box mt="2xl">Hero</Box>
+    <Box
+      height="100%"
+      width="100%"
+      overflow="hidden"
+      bg="url(/img/hero-illustration.svg)"
```
Comment (KipandaJr) [inline views/institutional/components/hero/index.tsx]: here too, webp

**Review by KipandaJr — CHANGES_REQUESTED** [review]
mobile version looks weird

**Review by KipandaJr — CHANGES_REQUESTED** [review]
When u free tell me to do a sair programming in war-room.

**Pull request merged**